### PR TITLE
add support of screenX and screenY properties in eventSimulator (closes #1577)

### LINKED ISSUE #1577: screenX and screenY properties are not supported when simulating event
connected with https://github.com/DevExpress/testcafe/issues/2325

### PR DESCRIPTION
**Pull request by AlexKamaev**

### CODE REVIEW

File: test/client/fixtures/sandbox/event/event-simulator-test.js [hunk 1]
```diff
@@ -217,6 +217,20 @@ test('event with coords (clientX, clientY)', function () {
     ok(raised);
 });
 
+test('event with coords (screenX, screenY)', function () {
+    var screenX = 10;
+    var screenY = 10;
+
+    domElement['onmousedown'] = function (e) {
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/event/event-simulator-test.js]: domElement.onmousedown = function (e) ...

File: test/client/fixtures/sandbox/event/event-simulator-test.js [hunk 2]
```diff
@@ -217,6 +217,20 @@ test('event with coords (clientX, clientY)', function () {
...
+        var ev = e || window.event;
```
Comment (miherlosev) [inline test/client/fixtures/sandbox/event/event-simulator-test.js]: I guess it is not necessary. Just use an `e` variable.

**Comment by AlexKamaev** [comment]
@testcafe-build-bot \retest

**Comment by AlexKamaev** [comment]
@testcafe-build-bot \retest

**Pull request merged**